assignments/prototypes.js

```diff
@@ -15,13 +15,30 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
-
+function GameObject(attrs){
+  this.createdAt = attrs.createdAt;
+  this.name = attrs.name;
+  this.dimensions = attrs.dimensions;
+}
+GameObject.prototype.destroy = function(remove){
+  console.log(`${this.name} was removed from the game`);
+}
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function CharacterStats(healthPoints){
+  GameObject.call(this, healthPoints);
+  this.damageAmount = healthPoints.damageAmount;
+}
+CharacterStats.prototype = Object.create (GameObject.prototype);
+CharacterStats.prototype.takeDamage = function(){
+  console.log(`${this.name} took damage`);
```

Comment:
I love seeing that you are checking your work before you return it. Shows that you care about quality.